i8_terminal/commands/screen/screen_search.py

```diff
@@ -59,6 +59,14 @@ def sort_by_tickers(df: pd.DataFrame, sorted_tickers: List[str]) -> pd.DataFrame
     return df
 
 
+def sort_df_include_periods(df: pd.DataFrame, metrics_include_periods: List[str]) -> pd.DataFrame:
+    column_orders = list(df.columns)
+    for metric_period in set(metrics_include_periods):
```

Comment:
This logic is confusing. What are you trying to do? The name of function is sort but there is no sorting. Also `metrics_include_periods` seems to only be metric names. Naming is very confusing.